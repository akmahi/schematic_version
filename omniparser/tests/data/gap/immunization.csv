"Row","Domain","Patient First Name","Patient Last Name","Patient DOB","External Patient ID","External Practice ID","Code","Name","Dictionary Name","Date Performed","Date Ordered","Date Due","Refused","Med Unnecessary RSN Code","Med Unnecessary RSN Name","Med Unnecessary RSN Dict","NPI","TIN","Provider Name","Administered Amount","Administered Unit","Vaccine Lot Number","Immu Manufacturer Code","Vendor","Source Type","PATIENT_ID","Remove Specified Immu Flag","VFC Eligibility Dict Name","VFC Eligibility Code"
"1","Immunizations","SIMONE","ABRAMS","19641214","3368664","Epic","100208","COVID-19 VACCINE (PFIZER)","","20210312","","","","","","","1053603076","","TUXHARI, SNEZHANA","","","EN6207","","","","AE3HBT4LP6ERD2F4CBO6QMLUPFCQHU5QZGYC2F5V","","",""
"2","Immunizations","SIMONE","ABRAMS","19641214","3368664","Epic","100150","INFLUENZA VACCINE, INJECTION, QUADRIVALENT (PRESERVATIVE-FREE)","","20201022","","","","","","","1053603076","","TUXHARI, SNEZHANA","","","UJ427AB","","","","AE3HBT4LP6ERD2F4CBO6QMLUPFCQHU5QZGYC2F5V","","",""
"3","Immunizations","SIMONE","ABRAMS","19641214","3368664","Epic","100150","INFLUENZA VACCINE, INJECTION, QUADRIVALENT (PRESERVATIVE-FREE)","","20171128","","","","","","","1053603076","","TUXHARI, SNEZHANA","","","JL59K","","","","AE3HBT4LP6ERD2F4CBO6QMLUPFCQHU5QZGYC2F5V","","",""
"4","Immunizations","SIMONE","ABRAMS","19641214","3368664","Epic","100150","INFLUENZA VACCINE, INJECTION, QUADRIVALENT (PRESERVATIVE-FREE)","","20181025","","","","","","","1053603076","","TUXHARI, SNEZHANA","","","UJ046AB","","","","AE3HBT4LP6ERD2F4CBO6QMLUPFCQHU5QZGYC2F5V","","",""
"5","Immunizations","SIMONE","ABRAMS","19641214","3368664","Epic","100208","COVID-19 VACCINE (PFIZER)","","20210402","","","","","","","1053603076","","TUXHARI, SNEZHANA","","","EW0150","","","","AE3HBT4LP6ERD2F4CBO6QMLUPFCQHU5QZGYC2F5V","","",""
"6","Immunizations","SIMONE","ABRAMS","19641214","3368664","Epic","100208","COVID-19 VACCINE (PFIZER)","","20211116","","","","","","","1053603076","","TUXHARI, SNEZHANA","","","FH8028","","","","AE3HBT4LP6ERD2F4CBO6QMLUPFCQHU5QZGYC2F5V","","",""
"7","Immunizations","SIMONE","ABRAMS","19641214","3368664","Epic","100150","INFLUENZA VACCINE, INJECTION, QUADRIVALENT (PRESERVATIVE-FREE)","","20170120","","","","","","","1053603076","","TUXHARI, SNEZHANA","","","UI623AA","","","","AE3HBT4LP6ERD2F4CBO6QMLUPFCQHU5QZGYC2F5V","","",""
"8","Immunizations","SIMONE","ABRAMS","19641214","3368664","Epic","100300","COVID-19 VACCINE (PFIZER), BIVALENT, PF, 30 MCG/0.3 ML","","20220910","","","","","","","1053603076","","TUXHARI, SNEZHANA","","","GH9702","","","","AE3HBT4LP6ERD2F4CBO6QMLUPFCQHU5QZGYC2F5V","","",""
"9","Immunizations","SIMONE","ABRAMS","19641214","3368664","Epic","100150","INFLUENZA VACCINE, INJECTION, QUADRIVALENT (PRESERVATIVE-FREE)","","20191122","","","","","","","1053603076","","TUXHARI, SNEZHANA","","","UJ305AB","","","","AE3HBT4LP6ERD2F4CBO6QMLUPFCQHU5QZGYC2F5V","","",""
"10","Immunizations","SIMONE","ABRAMS","19641214","3368664","Epic","100150","INFLUENZA VACCINE, INJECTION, QUADRIVALENT (PRESERVATIVE-FREE)","","20220118","","","","","","","1053603076","","TUXHARI, SNEZHANA","","","L494C","","","","AE3HBT4LP6ERD2F4CBO6QMLUPFCQHU5QZGYC2F5V","","",""
