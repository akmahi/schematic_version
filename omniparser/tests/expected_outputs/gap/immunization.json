[
	{
		"immunization": {
			"code": "100208",
			"date_performed": "20210312",
			"domain": "Immunizations",
			"name": "COVID-19 VACCINE (PFIZER)",
			"patient": {
				"dob": "19641214",
				"external_id": "3368664",
				"first_name": "SIMONE",
				"id": "AE3HBT4LP6ERD2F4CBO6QMLUPFCQHU5QZGYC2F5V",
				"last_name": "ABRAMS"
			},
			"practice": {
				"external_id": "Epic"
			},
			"provider": {
				"name": "TUXHARI, SNEZHANA",
				"npi": "1053603076"
			},
			"row": "1",
			"vaccine_lot_number": "EN6207"
		}
	},
	{
		"immunization": {
			"code": "100150",
			"date_performed": "20201022",
			"domain": "Immunizations",
			"name": "INFLUENZA VACCINE, INJECTION, QUADRIVALENT (PRESERVATIVE-FREE)",
			"patient": {
				"dob": "19641214",
				"external_id": "3368664",
				"first_name": "SIMONE",
				"id": "AE3HBT4LP6ERD2F4CBO6QMLUPFCQHU5QZGYC2F5V",
				"last_name": "ABRAMS"
			},
			"practice": {
				"external_id": "Epic"
			},
			"provider": {
				"name": "TUXHARI, SNEZHANA",
				"npi": "1053603076"
			},
			"row": "2",
			"vaccine_lot_number": "UJ427AB"
		}
	},
	{
		"immunization": {
			"code": "100150",
			"date_performed": "20171128",
			"domain": "Immunizations",
			"name": "INFLUENZA VACCINE, INJECTION, QUADRIVALENT (PRESERVATIVE-FREE)",
			"patient": {
				"dob": "19641214",
				"external_id": "3368664",
				"first_name": "SIMONE",
				"id": "AE3HBT4LP6ERD2F4CBO6QMLUPFCQHU5QZGYC2F5V",
				"last_name": "ABRAMS"
			},
			"practice": {
				"external_id": "Epic"
			},
			"provider": {
				"name": "TUXHARI, SNEZHANA",
				"npi": "1053603076"
			},
			"row": "3",
			"vaccine_lot_number": "JL59K"
		}
	},
	{
		"immunization": {
			"code": "100150",
			"date_performed": "20181025",
			"domain": "Immunizations",
			"name": "INFLUENZA VACCINE, INJECTION, QUADRIVALENT (PRESERVATIVE-FREE)",
			"patient": {
				"dob": "19641214",
				"external_id": "3368664",
				"first_name": "SIMONE",
				"id": "AE3HBT4LP6ERD2F4CBO6QMLUPFCQHU5QZGYC2F5V",
				"last_name": "ABRAMS"
			},
			"practice": {
				"external_id": "Epic"
			},
			"provider": {
				"name": "TUXHARI, SNEZHANA",
				"npi": "1053603076"
			},
			"row": "4",
			"vaccine_lot_number": "UJ046AB"
		}
	},
	{
		"immunization": {
			"code": "100208",
			"date_performed": "20210402",
			"domain": "Immunizations",
			"name": "COVID-19 VACCINE (PFIZER)",
			"patient": {
				"dob": "19641214",
				"external_id": "3368664",
				"first_name": "SIMONE",
				"id": "AE3HBT4LP6ERD2F4CBO6QMLUPFCQHU5QZGYC2F5V",
				"last_name": "ABRAMS"
			},
			"practice": {
				"external_id": "Epic"
			},
			"provider": {
				"name": "TUXHARI, SNEZHANA",
				"npi": "1053603076"
			},
			"row": "5",
			"vaccine_lot_number": "EW0150"
		}
	},
	{
		"immunization": {
			"code": "100208",
			"date_performed": "20211116",
			"domain": "Immunizations",
			"name": "COVID-19 VACCINE (PFIZER)",
			"patient": {
				"dob": "19641214",
				"external_id": "3368664",
				"first_name": "SIMONE",
				"id": "AE3HBT4LP6ERD2F4CBO6QMLUPFCQHU5QZGYC2F5V",
				"last_name": "ABRAMS"
			},
			"practice": {
				"external_id": "Epic"
			},
			"provider": {
				"name": "TUXHARI, SNEZHANA",
				"npi": "1053603076"
			},
			"row": "6",
			"vaccine_lot_number": "FH8028"
		}
	},
	{
		"immunization": {
			"code": "100150",
			"date_performed": "20170120",
			"domain": "Immunizations",
			"name": "INFLUENZA VACCINE, INJECTION, QUADRIVALENT (PRESERVATIVE-FREE)",
			"patient": {
				"dob": "19641214",
				"external_id": "3368664",
				"first_name": "SIMONE",
				"id": "AE3HBT4LP6ERD2F4CBO6QMLUPFCQHU5QZGYC2F5V",
				"last_name": "ABRAMS"
			},
			"practice": {
				"external_id": "Epic"
			},
			"provider": {
				"name": "TUXHARI, SNEZHANA",
				"npi": "1053603076"
			},
			"row": "7",
			"vaccine_lot_number": "UI623AA"
		}
	},
	{
		"immunization": {
			"code": "100300",
			"date_performed": "20220910",
			"domain": "Immunizations",
			"name": "COVID-19 VACCINE (PFIZER), BIVALENT, PF, 30 MCG/0.3 ML",
			"patient": {
				"dob": "19641214",
				"external_id": "3368664",
				"first_name": "SIMONE",
				"id": "AE3HBT4LP6ERD2F4CBO6QMLUPFCQHU5QZGYC2F5V",
				"last_name": "ABRAMS"
			},
			"practice": {
				"external_id": "Epic"
			},
			"provider": {
				"name": "TUXHARI, SNEZHANA",
				"npi": "1053603076"
			},
			"row": "8",
			"vaccine_lot_number": "GH9702"
		}
	},
	{
		"immunization": {
			"code": "100150",
			"date_performed": "20191122",
			"domain": "Immunizations",
			"name": "INFLUENZA VACCINE, INJECTION, QUADRIVALENT (PRESERVATIVE-FREE)",
			"patient": {
				"dob": "19641214",
				"external_id": "3368664",
				"first_name": "SIMONE",
				"id": "AE3HBT4LP6ERD2F4CBO6QMLUPFCQHU5QZGYC2F5V",
				"last_name": "ABRAMS"
			},
			"practice": {
				"external_id": "Epic"
			},
			"provider": {
				"name": "TUXHARI, SNEZHANA",
				"npi": "1053603076"
			},
			"row": "9",
			"vaccine_lot_number": "UJ305AB"
		}
	},
	{
		"immunization": {
			"code": "100150",
			"date_performed": "20220118",
			"domain": "Immunizations",
			"name": "INFLUENZA VACCINE, INJECTION, QUADRIVALENT (PRESERVATIVE-FREE)",
			"patient": {
				"dob": "19641214",
				"external_id": "3368664",
				"first_name": "SIMONE",
				"id": "AE3HBT4LP6ERD2F4CBO6QMLUPFCQHU5QZGYC2F5V",
				"last_name": "ABRAMS"
			},
			"practice": {
				"external_id": "Epic"
			},
			"provider": {
				"name": "TUXHARI, SNEZHANA",
				"npi": "1053603076"
			},
			"row": "10",
			"vaccine_lot_number": "L494C"
		}
	}
]
